library/globals.inc.php

```diff
@@ -2635,7 +2629,7 @@
         'ms' => xl('Malay'),
         'ml' => xl('Malayalam'),
         'mt' => xl('Maltese'),
-        'mi' => xl('Māori'),
+        'mi' => xl('Maori'),
         'mr' => xl('Marathi (Marāṭhī)'),
```

Comment:
Well, there are a lot more non-US keyboard characters in all that.  Any suggestions?